app/controllers/account_requests_controller.rb

```diff
@@ -18,10 +18,12 @@ def invalid_token; end
 
   def new
     @account_request = AccountRequest.new
+    @bank_selected
```

Comment:
This line doesn't actually do anything. Did you mean to write `@bank_selected = true` (or false)?

Reply:
So my line of thinking was that this was actually calling on the variable in the `bank_selected` variable within the private method I had created. I was under the impression that it was necessary to pass the value from the `create` controller action to the `new` controller action in Rails. However, as you can tell, my understanding of re-rendering operations was incorrect.

I have removed this from the #new controller action 🙌